src/sfctl/helps/deployment.py

```diff
@@ -0,0 +1,23 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -----------------------------------------------------------------------------
+
+"""Help documentation for managing Service Fabric Mesh Resources."""
+
+from knack.help_files import helps
+
+helps['mesh deployment create'] = """
+    type: command
+    short-summary: Creates a deployment of Service Fabric Mesh Resources
+    parameters:
+        - name: --input-yaml-files-path
+          type: string
+          short-summary: Comma seperated file paths of all the yaml files or the directory(recursive) which contain yaml files
+    examples:
+        - name: Consolidates and deploys all the resources to a cluster endpoint

```

Comment:
"to a cluster endpoint" sounds weird to me. Do you mean just that it adds it to the cluster set by sfctl cluster select?

Reply:
Changing this to "to cluster" . Let me know if you think we should change this to something other than this.